rust/noosphere-ucan/Cargo.toml

```diff
@@ -19,6 +19,10 @@ rust-version = "1.75.0"
 
 [features]
 default = []
+ed25519 = ["dep:ed25519-dalek"]
+rsa = ["dep:rsa", "dep:sha2"]
+p256 = ["dep:p256"]
+web-rsa = ["dep:rsa", "dep:sha2", "dep:js-sys", "dep:web-sys", "dep:wasm-bindgen", "dep:wasm-bindgen-futures"]
```

Comment:
Maybe should be `web-crypto-rsa`?

Reply:
I'm fine with either. Mildly prefer `web-crypto-rsa`.